code/datums/status_effects/buffs.dm

```diff
@@ -355,6 +355,42 @@
 /datum/status_effect/regenerative_core/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IGNOREDAMAGESLOWDOWN, id)
 
+/datum/status_effect/fleshmend
+	duration = -1
+	status_type = STATUS_EFFECT_REFRESH
+	tick_interval = 1 SECONDS
+	alert_type = null
+	var/tolerance = 1 // This diminishes the healing of fleshmend the higher it is.
+	var/instance_duration = 10 // in ticks
+	var/active_instances[0] // a list of integers, one for each remaining instance of fleshmend.
+	var/ticks = 0
+
+/datum/status_effect/fleshmend/on_apply()
+	tolerance += 1
+	active_instances += instance_duration // need to make sure this is a new instance and not just the existing instance variable
+	return TRUE
+
+/datum/status_effect/fleshmend/refresh()
+	tolerance += 1
+	active_instances += instance_duration
+	..()
+
+/datum/status_effect/fleshmend/tick()
+	if(active_instances.len >= 1)
```

Comment:
```suggestion
	if(length(active_instances))
```

Reply:
shouldn't this be "if(length(active_instances) >= 1)" ?

Reply:
It doesn't need to be, if length is 0 it fails, if length is any other number other than 0 it passes, which is the same as checking if the number if higher than or equal to 1